### PR TITLE
Deprecated mysqli_ping()

### PR DESCRIPTION
**Pull request by kamil-tekiela**

### CODE REVIEW

**Review by Girgias — APPROVED** [review]
Did the `thread_id()` change slip in due to some other PR?

**Comment by kamil-tekiela** [comment]
> Did the `thread_id()` change slip in due to some other PR?

I found it while making this PR and decided to include it in. Would you like me to commit it separately?

File: reference/mysqli/mysqli/ping.xml [hunk 1]
```diff
@@ -58,6 +58,31 @@
...
+     </row>
+    </thead>
+    <tbody>
+     <row>
+      <entry>8.4.0</entry>
+      <entry>
+       Both <methodname>mysqli::ping</methodname> and
+       <function>mysqli_ping</function> are now deprecated.
+       The <literal>reconnect</literal> feature has not been available
+       since PHP 8.2.0 making this function obsolete.
```
Comment (cmb69) [inline reference/mysqli/mysqli/ping.xml:78]:
```suggestion
       as of PHP 8.2.0, making this function obsolete.
```

**Review by cmb69 — COMMENTED** [review]
Thank you!

In my opinion the unrelated fix can be commited here (no need for a separate PR).

**Comment by Girgias** [comment]
> > Did the `thread_id()` change slip in due to some other PR?
> 
> I found it while making this PR and decided to include it in. Would you like me to commit it separately?

I don't mind, just wanted to make sure it was intentional :)

**Pull request merged**